tests/plugins/Sticky.test.ts

```diff
@@ -0,0 +1,33 @@
+import Sticky from '../../src/plugins/Sticky';
+import MockAdNetwork from '../../src/networks/Mock';
```

Comment:
Doesn't look like Noop or MockAdNetwork are being used. Can we delete?

Reply:
yes.